packages/components/ng-ovh-chart/src/directive.js

```diff
@@ -63,5 +63,26 @@ export default /* @ngInject */ () => ({
         }
       });
     };
+
+    this.$onDestroy = () => {
+      if (this.chartInstance) {
+        const pluginPrometheus = this.chartInstance['datasource-prometheus'];
+        if (!pluginPrometheus) {
+          this.chartInstance.destroy();
+          return;
+        }
```

Comment:
**nitpick:** This seems to be duoplicated https://github.com/ovh/manager/blob/d9641a04a91a2e09707e49a5080d821d96142376/packages/components/ng-ovh-chart/src/directive.js#L60-L64.
Can you remove `$scope..$on('destroy')....`?